app/assets/stylesheets/dm/pages/_visns.scss

```diff
@@ -171,4 +171,7 @@
       text-decoration: none !important;
     }
   }
+  .visn-tooltip-header{
+    vertical-align: top !important;
+  }
```

Comment:
I think we can use the USWDS class `text-top` to achieve this. We can remove this.